frontend/src/component/Workspace/ContentItem.jsx

```diff
@@ -75,8 +77,19 @@ class ContentItem extends React.Component {
             <FilenameWithBadges file={props} isTemplate={props.isTemplate} customClass='content__name' />
           </div>
 
+          <TimedEvent
+            customClass='content__lastModification'
+            operation={getRevisionTypeLabel(props.currentRevisionType, props.t)}
+            date={props.modified}
+            lang={props.lang}
+            author={{
+              publicName: props.lastModifier.public_name,
+              userId: props.lastModifier.user_id
```

Comment:
to avoid using snake case prooperty names, see comment in reducer workspaceContentList